scripts/setup

```diff
@@ -1,118 +1,33 @@
 #!/bin/bash
+
 set -e
 
 if [[ -n "${RF_DEBUG}" ]]; then
     set -x
 fi
 
-DIR="$(dirname "$0")"
-
-
 function usage() {
-
     echo -n \
-"Usage: $(basename "$0")
-Setup external project dependencies.
+         "Usage: $(basename "$0")
+Attempts to setup the project's development environment.
 "
 }
 
-
-function check_database() {
```

Comment:
Fixed in [4848c05](https://github.com/raster-foundry/raster-foundry/pull/2415/commits/4848c058a0c482dc4225968195be1c829dbb3038)